src/main/java/com/rocketden/main/model/Room.java

```diff
@@ -66,6 +68,7 @@
 
     @ManyToMany(fetch = FetchType.EAGER)
     @JoinColumn(name = "selectable_problem_id")
+    @Fetch(FetchMode.SUBSELECT)
```

Comment:
What does this do?

Reply:
ManyToMany eagerly fetched apparently by default does a [cartesian product](https://stackoverflow.com/questions/22224931/getting-duplicate-records-when-using-manytomany-annotation) causing duplicates, so this was necessary to fix that

Reply:
Got it, sounds good - I will note that the [SubSelect Mode](https://stackoverflow.com/questions/32984799/fetchmode-join-vs-subselect) could mean performance issues later in development, so we end up restructuring this in the future, but I think this works well for now.